src/AppBundle/Entity/City.php

```diff
@@ -40,6 +40,13 @@ class City
      */
     protected $connections;
 
+    /**
+     * @var string
+     *
+     * @ORM\Column(name="sender_email", type="string", nullable=true)
```

Comment:
I disagree with you. That property "name" have been added there for optional naming of database field. There might be some cases that a database field is NOT the same as the property in an Entity class. It's not redundancy. Also it's not that of a big deal having it there.